pkg/cmd/codespace/create.go

```diff
@@ -115,10 +120,81 @@ func (a *App) Create(ctx context.Context, opts createOptions) error {
 		Machine:            machine,
 		Location:           locationResult.Location,
 		IdleTimeoutMinutes: int(opts.idleTimeout.Minutes()),
+		PermissionsOptOut:  opts.permissionsOptOut,
 	})
 	a.StopProgressIndicator()
+
 	if err != nil {
-		return fmt.Errorf("error creating codespace: %w", err)
+		var aerr api.AcceptPermissionsRequiredError
```

Comment:
This `Create` method is already quite large, wdyt about moving this whole code path in something like `a.handleAdditionalPermissions(...)` ? and then doing:

```
...
if !errors.As(err, &aerr) || aerr.AllowPermissionsURL == "" {
			return fmt.Errorf("error creating codespace: %w", err)
		}

return a.handleAdditionalPermissions(...)
```

Reply:
@mislav suggested `--default-permissions` which I like as well